juju/testing/conn.go

```diff
@@ -67,7 +67,16 @@ import (
 	jujuversion "github.com/juju/juju/version"
 )
 
-const ControllerName = "kontroll"
+const (
+	ControllerName = "kontroll"
+
+	kubernetesSeriesName = "kubernetes"
```

Comment:
I wonder if we could get this const from juju/os; maybe something like a `(Must)GetOSFromSeries("kubernetes")`

Reply:
Ah, yes, let's make that change. Would clean up a lot of things.